server/dmain.cpp

```diff
@@ -47,11 +47,13 @@ int wmain(int argc, wchar_t* argv[], wchar_t* envp[]) {
 #ifndef FREEORION_DMAIN_KEEP_STACKTRACE
     try {
 #endif
-        GetOptionsDB().AddFlag('h', "help",         UserStringNop("OPTIONS_DB_HELP"),         false);
-        GetOptionsDB().AddFlag('v', "version",      UserStringNop("OPTIONS_DB_VERSION"),      false);
-        GetOptionsDB().AddFlag('s', "singleplayer", UserStringNop("OPTIONS_DB_SINGLEPLAYER"), false);
-        GetOptionsDB().AddFlag("hostless",          UserStringNop("OPTIONS_DB_HOSTLESS"),     false);
-        GetOptionsDB().AddFlag("skip-checksum",     UserStringNop("OPTIONS_DB_SKIP_CHECKSUM"), false);
+        GetOptionsDB().AddFlag('h', "help",          UserStringNop("OPTIONS_DB_HELP"),              false);
+        GetOptionsDB().AddFlag('v', "version",       UserStringNop("OPTIONS_DB_VERSION"),           false);
+        GetOptionsDB().AddFlag('s', "singleplayer",  UserStringNop("OPTIONS_DB_SINGLEPLAYER"),      false);
+        GetOptionsDB().AddFlag("hostless",           UserStringNop("OPTIONS_DB_HOSTLESS"),          false);
+        GetOptionsDB().AddFlag("skip-checksum",      UserStringNop("OPTIONS_DB_SKIP_CHECKSUM"),     false);
+        GetOptionsDB().Add<int>("mplobby-max-ai",    UserStringNop("OPTIONS_DB_MPLOBBY_MAX_AI"),   -1, Validator<int>());
```

Comment:
why not storable?

Reply:
I suppose if someone launch server without those limits he awaits to see default unlimited values not the stored ones.